strings/strings.c

```diff
@@ -9,7 +9,13 @@
 */
 int string_length(char *s)
 {
+    int length = 0;
 
+    while (*s != '\0') {
```

Comment:
You could also change the while loop to `while(s[length] != '\0')` and then only increment length.